From: Mathias Gibbens <gibmat@debian.org>
Description: Fix compilation with newer versions of cloudflare-cfssl library
Forwarded: https://github.com/notaryproject/notary/pull/1532
diff --git a/trustpinning/certs_test.go b/trustpinning/certs_test.go
index 4f5dc96f..606ee68e 100644
--- a/trustpinning/certs_test.go
+++ b/trustpinning/certs_test.go
@@ -74,7 +74,7 @@ func sampleCertChain(t *testing.T) *certChain {
 		// Create a simple CSR for the CA using the default CA validator and policy
 		req := &csr.CertificateRequest{
 			CN:         "docker.io/notary/root",
-			KeyRequest: csr.NewBasicKeyRequest(),
+			KeyRequest: csr.NewKeyRequest(),
 			CA:         &csr.CAConfig{},
 		}
 
